### DISCUSSION TITLE
shadowsocks-android支持防止重放吗？

### DISCUSSION THREAD

**Discussion by tianlichunhong** [shadowsocks/shadowsocks-android#2890] [Q&A]
服务器通过xray设置了防止重放，但是目前好像没有客户端支持。 shadowsocks-android支持防止重放吗？谢谢！

**Comment by Mygod** [comment]
no

**Reply by tianlichunhong** [reply]
在安卓termux上通过脚本+xray，实现了客户端防重放的功能。发现在双向防重放下，shadowsoks不再被GFW限速，速度明显提升（可以翻倍）。在没有设置防重放情况下，很容易被封端口，或者被QoS。期望能提供实现防重放的客户端。
